src/common/AllExperiments/Experiments_Live.cs

```diff
@@ -0,0 +1,126 @@
+
+            var vaSettings = new VariantAssignmentClientSettings
+            {
+                Endpoint = new Uri("https://default.exp-tas.com/exptas77/a7a397e7-6fbe-4f21-a4e9-3f542e4b000e-exppowertoys/api/v1/tas"),
+                EnableCaching = true,
+                ResponseCacheTime = TimeSpan.FromMinutes(5),
+            };
+
+            var vaClient = vaSettings.GetTreatmentAssignmentServiceClient();
+            var vaRequest = GetVariantAssignmentRequest();
+
+            try
+            {
+                var task = vaClient.GetVariantAssignmentsAsync(vaRequest);
+                TimeSpan ts = TimeSpan.FromMilliseconds(500);
+                if (task.Wait(ts))
+                {
+                    var result = task.Result;
+                    var allFeatureFlags = result.GetFeatureVariables();
+
+                    var assignmentContext = result.GetAssignmentContext();
+                    var featureNameSpace = allFeatureFlags[0].KeySegments[0];
+                    var featureFlag = allFeatureFlags[0].KeySegments[1];
+                    var featureFlagValue = allFeatureFlags[0].GetStringValue();
+
+                    if (featureFlagValue == "alternate")
+                    {
+                        IsExperiment = true;
+                    }
+
+                    PowerToysTelemetry.Log.WriteEvent(new OobeVariantAssignmentEvent() { AssignmentContext = assignmentContext, ClientID = AssignmentUnit });
+                }
+            }
+            catch (Exception ex)
+            {
+                Logger.LogError("Error getting variant assignments for experiment", ex);
+            }
+        }
+
+        public bool IsExperiment { get; set; }
+
+        private string? AssignmentUnit { get; set; }
+
+        private IVariantAssignmentRequest GetVariantAssignmentRequest()
+        {
+            string? clientID = string.Empty;
+
+            string workingDirectory = Environment.CurrentDirectory;
+            var exeDir = Directory.GetParent(workingDirectory)?.Parent?.Parent?.FullName;
+            string settingsPath = @"src\common\AllExperiments\settings.json";
+            if (exeDir != null)
+            {
+                string jsonFilePath = Path.Combine(exeDir, settingsPath);
+
+                string json = File.ReadAllText(jsonFilePath);
+                var jsonDictionary = JsonSerializer.Deserialize<Dictionary<string, object>>(json);
+                if (jsonDictionary != null)
+                {
+                    if (!jsonDictionary.ContainsKey("clientid"))
+                    {
+                        jsonDictionary.Add("clientid", string.Empty);
+                        jsonDictionary["clientid"] = Guid.NewGuid().ToString();
+                        string output = JsonSerializer.Serialize(jsonDictionary);
+                        File.WriteAllText(jsonFilePath, output);
+                    }
+
+                    clientID = jsonDictionary["clientid"]?.ToString();
```

Comment:
string "clientid" is used a lot here. I think it's worth saving it as a constant and using that constant instead of the string literal. It's too easy to misprint and not to notice it.